src/Avalonia.Controls.ColorPicker/ColorSpectrum/ColorSpectrum.Properties.cs

```diff
@@ -218,5 +226,21 @@ public ColorSpectrumShape Shape
             get => GetValue(ShapeProperty);
             set => SetValue(ShapeProperty, value);
         }
+
+        /// <summary>
+        /// Gets the third HSV color component that is NOT displayed by the spectrum.
+        /// This is automatically calculated from the <see cref="Components"/> property.
+        /// </summary>
+        /// <remarks>
+        /// This property should be used for any external color slider that represents the
+        /// third component of the color. Note that this property uses the generic
+        /// <see cref="ColorComponent"/> type instead of the more accurate <see cref="HsvComponent"/>
+        /// to allow direct usage by the generalized color sliders.
+        /// </remarks>
+        public ColorComponent ThirdComponent
```

Comment:
If it's supposed to be internal helper-property for the template, I personally like "[TemplateSettings](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.progressbar.templatesettings?view=winrt-22621#windows-ui-xaml-controls-progressbar-templatesettings)" pattern used in UWP.
But I don't think we need to introduce it here just for the one property.

Reply:
While this property is for the template right now I wouldn't consider that the only place it could ever be used. It actually fits in nicely with the Components property to make usage of this control with sliders easier -- it makes sense from an API standpoint.

Therefore, I wouldn't use the template settings pattern here at all.